flink-runtime/src/main/java/org/apache/flink/runtime/io/network/buffer/Buffer.java

```diff
@@ -55,9 +57,9 @@
 	boolean isBuffer();
 
 	/**
-	 * Tags this buffer to represent an event.
+	 * Tags this buffer to represent an event of the given type.
 	 */
-	void tagAsEvent();
+	void tagAsEvent(DataType dataType);
```

Comment:
It seems not clear whether the passed `dataType` must be an event type or not. So the method naming with argument seems not consistent, i prefer to naming the method as `tagBufferType`